pkg/api/v1/persistentvolume/util.go

```diff
@@ -129,6 +131,14 @@ func VisitPVSecretNames(pv *corev1.PersistentVolume, visitor Visitor) bool {
 				return false
 			}
 		}
+		if utilfeature.DefaultFeatureGate.Enabled(features.ExpandCSIVolumes) {
```

Comment:
no need to check the feature gate here. this field will always be nil until the feature is enabled, and if there's data in this field, the visitor should honor it